feelslikemonday/app/src/main/java/com/example/feelslikemonday/service/SortObjectDateTime.java

```diff
@@ -0,0 +1,17 @@
+package com.example.feelslikemonday.service;
+
+import com.example.feelslikemonday.model.followeeMoodEvent;
+import java.util.Comparator;
+
+/**
+ * This class contains a method that will compare mood events within a mood history
+ * It will sort them in reverse chronological order (based on the date and time the mood was posted)
+ */
+public class SortObjectDateTime implements Comparator<followeeMoodEvent>{
+
+    public int compare(followeeMoodEvent s1, followeeMoodEvent s2){
```

Comment:
Using a comparator is good. We can also implement comparable in the objects we want to compare

Reply:
you mean make FolloweeMoodEvent implements comparable?

Reply:
what would doing this change?

Reply:
We would have fewer objects and a bit less coupling. Comparators are good if we want to sort with multiple attributes, but both work in this case, so you don't have to change anything if you don't want to

Reply:
ok then if there is no need to change it I don't think I will because everything is already working nicely rn